tidy3d/packaging.py

```diff
@@ -195,31 +196,41 @@ def _fn(*args: Any, **kwargs: Any):
             tidy3d_extras["use_local_subpixel"] = False
             tidy3d_extras["mod"] = None
         else:
-            # first try to import the module
             if tidy3d_extras["mod"] is None:
-                try:
-                    import tidy3d_extras as tidy3d_extras_mod
-
-                except ImportError as exc:
-                    tidy3d_extras["mod"] = None
-                    tidy3d_extras["use_local_subpixel"] = False
-                    if preference is True:
+                tidy3d_extras["use_local_subpixel"] = False
+                tidy3d_extras["mod"] = None
```

Comment:
Maybe I miss something, but why is `mod` set to `None` again in this branch where `mod` is `None` to begin with?